src/plugins/share/public/components/tabs/export/export_content.tsx

```diff
@@ -218,19 +237,35 @@ const ExportContentUi = ({
     );
   };
 
+  const renderWarnings = (warning: { title: string; message: string }) => (
+    <>
+      <EuiSpacer size="m" />
+      <EuiCallOut color="warning" iconType="warning" title={warning.title}>
+        {warning.message}
+      </EuiCallOut>
+    </>
+  );
```

Comment:
Render warnings defined on the `ShareMenuItemV2`. This was inadvertently removed [here](https://github.com/elastic/kibana/pull/180406/files#diff-f32890b70ebccd874507683a4cc2bf273e7fe10db5bbba35b1b784e382dbab65L147-L164) where these warnings were never displayed.